src/parthenon_manager.cpp

```diff
@@ -84,9 +84,6 @@ ParthenonStatus ParthenonManager::ParthenonInit(int argc, char *argv[]) {
   }
 
   // Allow for user overrides to default Parthenon functions
-  if (app_input->SetFillDerivedFunctions != nullptr) {
-    SetFillDerivedFunctions = app_input->SetFillDerivedFunctions;
-  }
   if (app_input->ProcessProperties != nullptr) {
     ProcessProperties = app_input->ProcessProperties;
   }
```

Comment:
should `app_input` entirely be replaced with `params`? Or is it worth keeeping? I think that may be a longer discussion.

Reply:
I think this discussion has been had and the answer is no.